_std/macros/appearance.dm

```diff
@@ -0,0 +1,14 @@
+//non-hairstyle body accessory bitflags
+//corresponds to the color settings in user prefs
+#define DETAIL_1	(1<<0)	//
+#define DETAIL_2	(1<<1)	//
+#define DETAIL_3	(1<<2)	//
```

Comment:
I feel like these could have some prefix in their name to make them more distinct. If I randomly saw `DETAIL_1` in the code I wouldn't be sure what that means as the name is quite generic.

Reply:
Done!